### PR TITLE
Add instructions on how to get started with development

### LINKED ISSUE #3: Unable To Build - Missing Pods
I'm not familiar with Pods, should they be included in the repository or should I install Cocoa Pods?

### PR DESCRIPTION
**Pull request by wtimme**
This resolves #3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ Do you want to help testing pre-releases of OSM Surveyor?
 
 ![Screenshot of the map that displays annotations](screenshot.png?raw=true)
 
+# Getting started with development
+
+1. Install [Bundler][5]: `gem install bundler`
+2. Install the Gems (such as Cocoapods and fastlane): `bundle install`
+3. Install the dependencies using Cocoapods: `bundle exec pod install`
+4. Open the workspace (`OMSSurveyor.xcworkspace`)
```
Comment (pqhf5kd) [inline README.md:27]: 5. Open the workspace

File: README.md [hunk 2]
```diff
@@ -12,6 +12,13 @@ Do you want to help testing pre-releases of OSM Surveyor?
 
 ![Screenshot of the map that displays annotations](screenshot.png?raw=true)
 
+# Getting started with development
+
+1. Install [Bundler][5]: `gem install bundler`
+2. Install the Gems (such as Cocoapods and fastlane): `bundle install`
+3. Install the dependencies using Cocoapods: `bundle exec pod install`
```
Comment (pqhf5kd) [inline README.md:26]: 4. `bundle exec pod install` will installed the dependencies from the existing Podfile

File: README.md [hunk 3]
```diff
@@ -12,6 +12,13 @@ Do you want to help testing pre-releases of OSM Surveyor?
 
 ![Screenshot of the map that displays annotations](screenshot.png?raw=true)
 
+# Getting started with development
+
+1. Install [Bundler][5]: `gem install bundler`
+2. Install the Gems (such as Cocoapods and fastlane): `bundle install`
```
Comment (pqhf5kd) [inline README.md:25]: 2. CD to project folder. 
3. Install the Gems (such as Cocoapods and fastlane): `bundle install`. 'bundle install' will install the Gems needed from the existing Gemfile

File: README.md (hunk 2 above)
Comment (wtimme) [reply README.md:26]: Thanks for the feedback!  I've added a paragraph above the list that points out the `Podfile`.

File: README.md (hunk 3 above)
Comment (wtimme) [reply README.md:25]: Thanks for the feedback! You are correct: People who are not familiar with the Terminal might forget that second step, cd'ing into the project folder. I've added a paragraph above the list to point that out.

**Pull request merged**